contracts/digg-core/UFragments.sol

```diff
@@ -302,4 +304,17 @@ contract UFragments is ERC20Detailed, Ownable {
         emit Approval(msg.sender, spender, _allowedFragments[msg.sender][spender]);
         return true;
     }
+
+    /**
+     * @notice Mints the reimbursement for remDIGG one time, directly to dev multisig
+     */
+    function mintToDevMsig() external onlyOwner {
```

Comment:
What about if we use a more specific function name to avoid misinterpretations from users looking from Etherscan. Something like: `oneTimeMint()` or `oneTimeRemDiggMint()`. Maybe I am overthinking this?